mrmap/quality/models.py

```diff
@@ -37,7 +30,7 @@ class ConformityCheckConfiguration(UuidPk):
     def __str__(self):
         return self.name
 
-    def is_allowed_type(self, metadata: Metadata):
+    def is_allowed_type(self, metadata: DatasetMetadata):
```

Comment:
we shall not bind DatasetMetadata as param type here, cause we got different types in future.